cmake/modules/platform/toolcfg/gnu.cmake

```diff
@@ -69,6 +69,11 @@ if(OMR_ARCH_X86)
 	endif()
 endif()
 
+if(OMR_OS_LINUX)
+	list(APPEND OMR_PLATFORM_EXE_LINKER_OPTIONS " -Wl,-z,noexecstack")
+	list(APPEND OMR_PLATFORM_SHARED_LINKER_OPTIONS " -Wl,-z,noexecstack")
```

Comment:
I don't think we want the leading space in the string literals (the option needs to start with a dash, not a space).